### PR TITLE
Added Kubernetes-ja

### LINKED ISSUE #2596: [ja] Localize Kubernetes
Localize https://glossary.cncf.io/kubernetes/

### LINKED ISSUE #2597: [ja] Localize Cluster
Localize https://glossary.cncf.io/cluster/

### PR DESCRIPTION
**Pull request by aryasoni98**
### Describe your changes
Localize Kubernetes in Japan 🇯🇵 

### Related issue number or link (ex: `resolves #issue-number`)
resolve #2596, #2597

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by aryasoni98** [comment]
@inductor @naonishijima @kaitoii11 @yuichi-nakamura Can you please review the PR.

File: content/ja/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+従来の [コードとしてのインフラストラクチャ](/infra Structure-as-code/) ツールと同様に、Kubernetes は自動化に役立ちますが、コンテナと連携するという利点があります。
+コンテナーは、[virtual](/virtual-machine/) や物理マシンよりも構成のドリフトに対する耐性が高くなります。
+
+さらに、Kubernetes は宣言的に動作します。つまり、オペレーターがマシンに何かを行う方法を指示するのではなく、インフラストラクチャがどうあるべきかを通常はマニフェスト ファイル (YAML など) として記述します。
+その後、Kubernetes が「方法」を処理します。
+その結果、Kubernetes はコードとしてのインフラストラクチャと非常に互換性が高くなります。
+
+Kubernetes は [自己修復](/self-healing/) も行います。
+クラスターの実際の状態は、オペレーターの望ましい状態と常に一致します。
+Kubernetes がマニフェスト ファイルに記述されている内容からの逸脱を検出した場合、Kubernetes コントローラーが起動して修正します。
```
Comment (naonishijima) [inline content/ja/kubernetes.md:34]: 「Kubernetes コントローラーが起動して修正します。」Kubernetesコントローラーは常時起動しているので、起動という言葉よりも「作動」等の言葉が合うと思いました。

**Review by naonishijima — COMMENTED** [review]
Thank you for your interest. However, I felt that you needed to become more familiar with the Japanese language. If you are still interested, I would appreciate it if you could deal with the comments I made.

**Comment by aryasoni98** [comment]
Yes, i'm interested. 
Anything i need to update on pr

**Comment by kaitoii11** [comment]
Thanks for the PR. However, I think we should not be moving forward as we have established in the past that the localization should be done by someone who is fluent in Japanese.
 
https://github.com/cncf/glossary/pull/2140#issuecomment-1610418343

**Comment by seokho-son** [comment]
Hi @aryasoni98, thank you for your contribution and interest in the Glossary project.

I'm one of the maintainers of this repository. As @kaitoii11 and @naonishijima have already pointed out, reviewing localization PRs requires considerable effort. 

Therefore, we need high-quality translations and localizations of the proposed content. In this context, the review process becomes challenging if the reviewers feel that the content does not meet the required quality standards, especially if it's not a proposal from a contributor who is fluent in the language of localization.
(also, we don't accept a careless PR by direct machine translation.)

I encourage you to explore other areas of contribution that you might be more comfortable with. Why not check out the contribution section in your language?
Thank you again for your efforts. :)

@kaitoii11 @naonishijima, please feel free to either close this PR or continue with it as you see fit.

**Pull request closed without merging**